wicket-core/src/main/java/org/apache/wicket/ajax/res/js/wicket-ajax-jquery.js

```diff
@@ -2117,11 +2117,11 @@
 						// create link element
 						var css = Wicket.Head.createElement("link");
 
-						// copy required attributes
-						css.id = node.getAttribute("id");
-						css.rel = node.getAttribute("rel");
-						css.href = node.getAttribute("href");
-						css.type = node.getAttribute("type");
+						// copy supplied attributes only.
+						for (var attrIdx = 0; attrIdx < node.attributes.length; attrIdx++) {
+							var attr = node.attributes[attrIdx];
+							css.setAttribute(attr.name, attr.value)
+						}
```

Comment:
What is the advantage of using jQuery over direct dom manipulation? 
IE 5.5 is dead so using element.attributes is fine?
https://developer.mozilla.org/en-US/docs/Web/API/Element/attributes